docs/scheduler.md

```diff
@@ -207,7 +207,7 @@ You can modify the configuration for data plane pods as follows:
 
 **Update the NODE_SELECTOR for storage target and storage replica**
 
-You have to add following environmental parameters in your storage class before provisioning volume. You can provision OpenEBS storage target pod to your application deployed namespace.
+You have to add following environmental parameters in your storage class before provisioning volume. You can provision OpenEBS storage target pod to your application deployed namespace. So that application pods can communicate with storage target pod easily.
```

Comment:
You must add the following environmental parameters....provisioning a volume. You can provision an OpenEBS storage target pod to the namespace where your application is deployed so that the application pods can communicate easily with the storage target pod.